src/lib/threadpool.c

```diff
@@ -128,6 +130,13 @@ struct pool_impl {
 };
 /* clang-format on */
 
+/* Callback does not allow data passing, we use a static variable to report
+ * errors back. */
+static int thread_key_create_err = 0;
+static void thread_key_create(void) {
+	thread_key_create_err = uv_key_create(&thread_identifier_key);
+}
```

Comment:
`PRE(thread_key_create_err == 0)`?

Reply:
Should be always that way because of static value and `uv_once` only calling this block once per execution.

Reply:
Yeah, more for documentation/sanity checking. Thanks!